### DISCUSSION TITLE
shared-buckets, my-private-bucket, my-public-bucket are all empty

### DISCUSSION THREAD

**Discussion by mccabete** [MAAP-Project/Community#799] [Platform] [answered]
In the new ADE, when I navigate to or `ls` shared-buckets, my-private-bucket, or  my-public-bucket, they all list as empty. I assumed this was similar to the `transport endpoint not connected` error I have seen in the past, but even after resrting my workspace and starting a fresh one, I am still getting empty paths. 

Any suggestions?

**Comment by bsatoriu** [comment, marked as answer]
@mccabete the bug is caused by a devfile reference to the just-decommissioned mas.ops.maap-project.org docker registry: https://github.com/MAAP-Project/maap-workspaces/blob/v3.1.0/devfiles/pangeo/devfile/devfile.yaml#L59

To workaround this issue until this bug is fixed, simply update the devfile image value referenced above with `mas.maap-project.org/root/che-sidecar-s3fs:main`. The devfile can be edited by clicking the "configure workspace" setting icon in the ADE "Workspaces" tab.

**Reply by mccabete** [reply]
@bsatoriu  when I swapped the image to `mas.maap-project.org/root/che-sidecar-s3fs:main`, I get the error below. I am able to load workspaces with the old image. 

<img width="826" alt="image" src="https://github.com/MAAP-Project/Community/assets/21266199/206aa838-ff95-4987-aba5-e9fec91d49e7">

**Reply by bsatoriu** [reply]
@mccabete it looks like you replaced your workspace image with the s3fs image. If you look at the file I linked to in my last comment, the s3fs image is on line 59. That is the image that needs to be updated. The workspace image on line 34 should stay the same.

**Reply by mccabete** [reply]
woops! Thanks for that catch.

**Comment by bsatoriu** [comment]
Dev ticket: https://github.com/MAAP-Project/Community/issues/800
